### PR TITLE
[OSCI] Add ctsai21 as an author 

### PR DESCRIPTION
**Pull request by ctsai21**
### Description
Add ctsai21 as an author
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/ctsai.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Cho Tsai
+short_name: ctsai
+twitter:
```
Comment (ashwin-pc) [inline _authors/ctsai.markdown:4]: Can you remove this line if you are not using it?

**Review by ashwin-pc — CHANGES_REQUESTED** [review]
@ctsai21  can you move the file to _authors/osci_2023? Sorry there was a mistake in my instructions. Apologies for that.

File: _authors/ctsai.markdown (hunk 1 above)
Comment (ctsai21) [reply _authors/ctsai.markdown]: Moved the file and removed the line!

**Review by dtaivpp — COMMENTED** [review]
Looks great!

**Review by dtaivpp — APPROVED** [review]

**Pull request merged**